python/friesian/example/recall/generate_test_data.py

```diff
@@ -175,8 +176,8 @@ def generate_data(args):
 
     parser.add_argument('--verbose', type=bool, default=True,
                         help='whether print more detail information')
-    parser.add_argument('--use_spark', type=bool, default=True,
-                        help='whether to use spark to generate vector embeddings')
+    parser.add_argument('--use_spark', action='store_true',
+                        help='Use spark to generate vector embeddings')
```

Comment:
store_true will default to be False?

Reply:
yes, it will be False

Reply:
Then why not set default to be True?

Reply:
Maybe we should use stroe_false and change the name to not_use_spack, then the default will be True